### PR TITLE
Samus review rpip62

### PR DESCRIPTION
**Pull request by orangesamus**
Mostly accounting for rpip63 specifying the scrub penalty change. Some additional minor feedback for wording/typos

### CODE REVIEW

File: RPIPs/RPIP-62.md [hunk 1]
```diff
@@ -8,27 +8,27 @@ status: Draft
 type: Protocol
 category: Core
 created: 2024-07-25
+requires: rpip-63
```
Comment (haloooloolo) [inline RPIPs/RPIP-62.md:11]:
```suggestion
requires: 63
```

Comment (haloooloolo) [reply RPIPs/RPIP-62.md:11]: I think this should just be the RPIP number for the website to generate a correct link.

File: RPIPs/RPIP-62.md [hunk 2]
```diff
@@ -8,27 +8,27 @@ status: Draft
...
 ---
 
 ## Abstract
 This proposal aims to improve RPL tokenomics in the short term before the changes of [RPIP-49](RPIP-49.md) can be implemented.
-Minipools can be created without a minimum RPL requirement and at 5% contract commission. A temporary (until after [Saturn 1](RPIP-55.md)) dynamic commission boost beyond this value is introduced. Total dynamic commission starts at 10% and scales linearly with RPL stake up to 14% at 10% of borrowed ETH.
-The cliff for RPL rewards is <explicitly maintained at 10% | removed by extending rewards linearly below 10%> of borrowed ETH. Scrub penalties are changed to be taken out of the node operator's ETH bond instead of slashing RPL.
+Minipools can be created without a minimum RPL requirement and at 5% contract commission. A temporary (until after [Saturn 1](RPIP-55.md)) dynamic commission boost beyond this value is introduced. Total dynamic commission starts at 10% for zero RPL staked, and scales linearly up to a max of 14% for RPL staked positions worth at least 10% of borrowed ETH.
```
Comment (haloooloolo) [inline RPIPs/RPIP-62.md:16]:
```suggestion
Minipools can be created without a minimum RPL requirement and at 5% contract commission. A temporary (until after [Saturn 1](RPIP-55.md)) dynamic commission boost beyond this value is introduced. Total dynamic commission starts at 10% for zero RPL staked, and scales linearly up to a max of 14% for staked RPL positions worth at least 10% of borrowed ETH.
```

Comment (haloooloolo) [reply RPIPs/RPIP-62.md:16]: Definitely an improvement over the current version. Switching it to "staked RPL position" from "RPL staked position" makes it sound a bit less awkward to me.

File: RPIPs/RPIP-62.md [hunk 3]
```diff
@@ -8,27 +8,27 @@ status: Draft
...
 Therefore, node operation is made more attractive by allowing minipool creation without RPL < | and removing the cliff for RPL rewards>.
 Contract commission for these new ETH-only minipools is kept less attractive than megapool validators under [Saturn 1](RPIP-55.md) to encourage migration once dynamic commission is disabled.
-In the interest of acting fast, this proposal minimizes smart contract changes. The suggested parameter changes can be enacted immediately after the vote passes and changes to the scrub penalty are implemented. The dynamic commission requires reward tree spec changes, which would be rolled out alongside the parameter changes or shortly thereafter.
+In the interest of acting fast, this proposal minimizes smart contract changes. The suggested parameter changes can be enacted immediately after the vote passes (though this rpip does require changing the scrub penalty to ETH instead of RPL, which is planned for implementation in the Houston Hotfix as specified in [RPIP-63](RPIP-63.md)). The dynamic commission requires reward tree spec changes, which would be rolled out alongside the parameter changes or shortly thereafter.
```
Comment (haloooloolo) [inline RPIPs/RPIP-62.md:25]:
```suggestion
In the interest of acting fast, this proposal minimizes smart contract changes. The suggested parameter changes can be enacted immediately after the vote passes and [RPIP-63](RPIP-63.md) is implemented. The dynamic commission requires reward tree spec changes, which would be rolled out alongside the parameter changes or shortly thereafter.
```

Comment (haloooloolo) [reply RPIPs/RPIP-62.md:25]: There's also a dependency on the division by zero fix, so maybe just simplify this a bit? Removed the second closing parenthesis.

**Review by haloooloolo — COMMENTED** [review]
Thanks!

**Review by Valdorff — APPROVED** [review]
This looks good to me. Not hitting merge button cuz of possible link to open discussions in https://github.com/rocket-pool/RPIPs/pull/291

**Pull request merged**